examples/remote-sensing/plot_clustering.py

```diff
@@ -0,0 +1,373 @@
+"""
+====================================================================
+Remote sensing: Clustering on hyperspectral/sar images with Riemannian
+geometry
+====================================================================
+
+This example compares clustering pipelines based on covariance matrices for
+hyperspectral/sar data custering.
+"""
+
+# Author: Ammar Mian
+
+import os
+import urllib.request
+import numpy as np
+import numpy.linalg as la
+from numpy.typing import ArrayLike
+import matplotlib.pyplot as plt
+
+from sklearn.pipeline import Pipeline
+from pyriemann.estimation import Covariances
+from pyriemann.clustering import Kmeans
+from scipy.io import loadmat
+from typing import Tuple, Dict
+
+from numpy.lib.stride_tricks import sliding_window_view
+from math import ceil 
+
+from sklearn.base import BaseEstimator, TransformerMixin
+from numpy.typing import ArrayLike
+
```

Comment:
You can use a #-line to separate imports from class definition.
```suggestion

###############################################################################
```